provision-dns-server.sh

```diff
@@ -1,10 +1,11 @@
-#!/bin/bash
+#!/bin/bash -ex
 source /vagrant/lib.sh
 
 pandora_ip_address="${1:-10.10.0.2}"; shift || true
 pandora_domain="${1:-pandora.k0s.test}"; shift || true
 k0s_domain="$(echo -n "$pandora_domain" | sed -E 's,^[a-z0-9-]+\.(.+),\1,g')"
 default_dns_resolver="$(resolvectl status | awk '/DNS Servers: /{print $3}')" # recurse queries through the default vagrant environment DNS server.
+[ -n "${default_dns_resolver}" ] || default_dns_resolver=8.8.8.8
```

Comment:
why is this required? if you are running this file multiple times, it was not made for that use-case.